src/App.tsx

```diff
@@ -8,7 +8,32 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodosStatus } from './types/TodoStatus';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [status, setStatus] = useState<TodosStatus>(TodosStatus.All);
+
+  useEffect(() => {
+    getTodos().then(todosFromServer => setTodos(todosFromServer));
+  }, []);
+
+  const filteredTodos = todos.filter(todo => {
+    const active = status === 'active' && todo.completed;
+    const completed = status === 'completed' && !todo.completed;
```

Comment:
```suggestion
  const filteredTodos = todos.filter(todo => {
    const active = status === 'active' && todo.completed;
    const completed = status === 'completed' && !todo.completed;
```

use parentheses for clarity